### PR TITLE
EPMRPP-38894 [5.0.1] React: 'Shared by me' icon is shown for filters 'shared by another user'fixed.

### PR DESCRIPTION
**Pull request by tritseratops**
…dded new filter icons. Updated behavior of newly saved filter also.

### CODE REVIEW

File: app/src/pages/inside/launchesPage/launchesPage.jsx [hunk 1]
```diff
@@ -653,6 +653,7 @@ export class LaunchesPage extends Component {
                   onResetFilter={onResetFilter}
                   onChangeSorting={this.handleChangeSorting}
                   sortingString={sortingString}
+                  userId={userId}
```
Comment (DiscoElevator) [inline app/src/pages/inside/launchesPage/launchesPage.jsx]: Looks like it's not used in `LaunchFiltersToolbar`.

File: app/src/pages/inside/common/launchFiltersToolbar/filterList/filterList.jsx [hunk 2]
```diff
@@ -12,6 +13,8 @@ export const FilterList = ({
   unsavedFilterIds,
   onSelectFilter,
   onRemoveFilter,
+  // userId,
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/launchFiltersToolbar/filterList/filterList.jsx]: Do you still need this?

File: app/src/pages/inside/launchesPage/launchesPage.jsx (hunk 1 above)
Comment (tritseratops) [reply app/src/pages/inside/launchesPage/launchesPage.jsx]: let me check

File: app/src/pages/inside/common/launchFiltersToolbar/filterList/filterList.jsx (hunk 2 above)
Comment (tritseratops) [reply app/src/pages/inside/common/launchFiltersToolbar/filterList/filterList.jsx]: seems no, I will delete it.

**Comment by tritseratops** [comment]
Сreated new pull request with all of the changes, because some how this ended with 2 commits.
New pull request
https://github.com/reportportal/service-ui/pull/1706

**Pull request closed without merging**